hawc/__init__.py

```diff
@@ -1,9 +1,15 @@
 import os
+import platform
 import sys
 
 
 def manage():
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "hawc.main.settings.dev")
     from django.core.management import execute_from_command_line
 
+    # Setuptools generate entry point scripts with "-script.py" appended
+    # to the end of them in Windows
+    if platform.system() != "Windows":
```

Comment:
shouldn't that be a `==`, not a `!=`?

Reply:
It should! Thank you for catching that, fixed.